README.md

```diff
@@ -137,11 +137,11 @@ The first example shows the simplest way to use `clap`, by defining a struct. If
 //
 // This example demonstrates clap's full 'custom derive' style of creating arguments which is the
 // simplest method of use, but sacrifices some flexibility.
-use clap::{AppSettings, Clap};
+use clap::{AppSettings, Parser};
```

Comment:
For some reason, CI isn't running against this PR.  This needs to be resolved before merging

Reply:
CI is running